tools.yaml

```diff
@@ -34,3 +34,321 @@ batman.r-forge.r-project.org:
+  
+stat.ethz.ch/R-manual/R-devel/library/MASS/html/lda.html:
+  name: Linear Discriminant Analysis
+  description: |
+    "Linear discriminant analysis."
+  url: https://stat.ethz.ch/R-manual/R-devel/library/MASS/html/lda.html
+  author:
+  license:
+    kind: GNU General Public License
+    url: https://www.r-project.org/Licenses/GPL-2
+
+cran.r-project.org/web/packages/muma/index.html:
+  name: Metabolomics Univariate and Multivariate Analysis
+  description: |
+    "Preprocessing of high-throughput data (normalization and scalings)"
+  url: https://cran.r-project.org/web/packages/muma/index.html
+  author: 
+    - name: Edoardo Gaude
+      email: egaude541@gmail.com
+    - name: Francesca Chignola
+    - name: Dimitrios Spiliotopoulos
+    - name: Silvia Mari
+    - name: Andrea Spitaleri
+    - name: Michela Ghitti
+  license:
+    kind: GNU GENERAL PUBLIC LICENSE
+    url: https://cran.r-project.org/web/licenses/GPL-3
+  
+stat.ethz.ch/R-manual/R-patched/library/stats/html/prcomp.html:
+  name: Principal Components Analysis
+  description: |
+    "Performs a principal components analysis on the given data matrix and returns the results as an object of class prcomp."
+  url: https://stat.ethz.ch/R-manual/R-patched/library/stats/html/prcomp.html
+  author: 
+  license:
+    kind: GNU General Public License
+    url: https://www.r-project.org/Licenses/GPL-2
+
+cran.cnr.berkeley.edu/:
+  name: The Comprehensive R Archive Network
+  description: >
+    "freely available language and environment for statistical computing and graphics which provides a
+    wide variety of statistical and graphical techniques."
+  url: http://cran.cnr.berkeley.edu/
+  author: 
+    - name:  The College of Natural Resources at University of California, Berkeley
+  license:
+    kind: GNU GENERAL PUBLIC LICENSE
+    url: https://www.r-project.org/Licenses/GPL-2
+
+# odd site 
+3omics.cmdm.tw:
+  name: 3omics
+  description: |
+    "Web tool for visualizing and rapidly integrating multiple inter- or intra-transcriptomic, proteomic, and metabolomic human data."
+  url: https://3omics.cmdm.tw
+  author:
+    - name: MDDLab
+  license: 
+    kind: 
+    url:
+
+# paid product
+acdlabs.com/products/adh/spectrusprocessor/index.php:
+  name: ACD/ NMR Processor Academic Edition
+  description: |
+    "Free NMR processing software for academic and non-commercial use only"
+  url: https://www.acdlabs.com/products/adh/spectrusprocessor/index.php
+  author:
+    - name: ACD/Labs
+  license: 
+    kind: 
+    url:
+
+CHEMDATA.NIST.GOV:
+  name: Mass Spectrometry Data Center
+  description: >
+    "Mass spectral libraries and data analysis tools include AMDIS 
+    (Automated Mass Spectral Deconvolution and Identification System for GC/MS) 
+    and MS Interpreter (for fragmentation analysis)."
+  url: https://chemdata.nist.gov
+  author:
+    - name: National Institute of Standards and Technology 
+      email: massspec@nist.gov
+  license: 
+    kind: open ? 
```

Comment:
Don't guess. If you didn't find a license, just omit these as comments like `open ?` will appear verbatim on the  website.